modular_skyrat/modules/reagent_forging/code/anvil.dm

```diff
@@ -6,6 +6,7 @@
 
 	anchored = TRUE
 	density = TRUE
+	///if on the mining zlevel, it is primitive and has a different icon
```

Comment:
IMO, this should instead be based on materials used, upgrades, if its pre-mapped to be high-tech, etc. It always felt weird for the station to be able to make seemingly high-tech stuff with the exact same materials as the primitive ones.
Just an irk though